rpc_proxy.go

```diff
@@ -537,7 +537,7 @@ func (p *rpcProxy) basicAuthToMacaroon(basicAuth, requestURI string,
 		}
 
 	case isLitURI(requestURI):
-		return EmptyMacaroonBytes, nil
+		macPath = p.cfg.MacaroonPath
```

Comment:
nit: remove the now unused variable `EmptyMacaroonBytes`.

Reply:
cool. Removed in next commit since it is still used in litcli at this stage